e2e/tests/interchain_accounts/base_test.go

```diff
@@ -26,6 +26,13 @@ import (
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
 )
 
+// orderMapping is a mapping from channel ordering to the string representation of the ordering.
+// the representation can be different depending on the relayer implmentation.
```

Comment:
bummer :sweat_smile:

Reply:
this must be something to do with how the enum value is serialized with JSON or something maybe? 

So `channeltypes.UNORDERED` evaluates to `ORDER_UNORDERED` in normal go code but here one of the relayers returns it as `Unordered`? is it hermes?